src/ae/oracle.js

```diff
@@ -76,9 +80,15 @@ function pollForQueries (oracleId, onQuery, { interval = 5000 } = {}) {
     if (queries.length) onQuery(queries)
   }
 
-  checkNewQueries()
-  const intervalId = setInterval(checkNewQueries, interval)
-  return () => clearInterval(intervalId)
+  let stopped
+  (async () => {
+    while (!stopped) { // eslint-disable-line no-unmodified-loop-condition
+      // TODO: allow to handle this error somehow
```

Comment:
still an open todo?

Reply:
@davidyuk  is now displaying the error on the console. TODO is open but I don't think it needs immediate attention.

Reply:
best way is throwing a custom error type.

Reply:
ok :)